packages/rspack-test-tools/tests/hookCases/javascriptModulesPlugin#chunkHash/update-hash/hooks.snap.txt

```diff
@@ -38,38 +38,38 @@ Array [
       "instancesPool": Array [],
       "mem": Object {
         "data": Array [
-          51,
-          49,
-          100,
-          54,
-          99,
-          102,
-          101,
-          48,
-          100,
-          49,
-          54,
-          97,
-          101,
-          57,
-          51,
-          49,
-          98,
-          55,
-          51,
-          99,
-          53,
-          57,
-          100,
-          55,
-          101,
-          48,
-          99,
-          48,
-          56,
-          57,
-          99,
-          48,
+          0,
```

Comment:
@ahabhgk Can you help to review the snapshot changes? I'm not sure what `hash.mem.data' means here.

Reply:
Related to the wasm instance pool, but the instance pool shouldn't affect the hash result, so the best way is adjust our test tool to snapshot the hash result instead of the hash inner structure layout

Reply:
Agreed, we can improve the test tool in another PR. The snapshot is now restored after fixing the instance pool.